Classes/Neos/Neos/Ui/FlowQueryOperations/FilteredChildrenOperation.php

```diff
@@ -0,0 +1,78 @@
+<?php
+namespace Neos\Neos\Ui\FlowQueryOperations;
+
+/*
+ * This file is part of the Neos.Neos.Ui package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Eel\FlowQuery\FizzleParser;
+use Neos\Eel\FlowQuery\FlowQuery;
+use Neos\Eel\FlowQuery\Operations\AbstractOperation;
+use Neos\Flow\Annotations as Flow;
+use Neos\ContentRepository\Domain\Model\NodeInterface;
+
+/**
+ * "children" operation working on ContentRepository nodes. It iterates over all
+ * context elements and returns all child nodes or only those matching
+ * the filter expression specified as optional argument.
+ */
+class FilteredChildrenOperation extends AbstractOperation
+{
+    /**
+     * {@inheritdoc}
+     *
+     * @var string
+     */
+    protected static $shortName = 'filteredChildren';
```

Comment:
I'd call this "neos_ui_filteredChildren", to ensure this is not accidentally called by other people